test/e2e/testdata/magiccatalog/fbc_updated.yaml

```diff
@@ -0,0 +1,42 @@
+---
+schema: olm.package
+name: packageA
+defaultChannel: stable
+---
+schema: olm.channel
+package: packageA
+name: stable
+entries:
+  - name: busybox.v1.0.0
+  - name: busybox.v2.0.0
+    replaces: busybox.v1.0.0
+---
+schema: olm.bundle
+name: busybox.v1.0.0
+package: packageA
+image: quay.io/olmtest/busybox-bundle:1.0.0
+properties:
+  - type: olm.gvk
+    value:
+      group: example.com
+      kind: TestA
+      version: v1alpha1
+  - type: olm.package
+    value:
+      packageName: packageA
+      version: 1.0.0
```

Comment:
Note: I realize I can probably reduce this file size by removing this olm.bundle channel entry entirely and this would still be a valid FBC.